src/app/app.service.ts

```diff
@@ -67,20 +70,20 @@ export class AppService {
      */
     public static async initAgent(app: INestApplication) {
         const agentService = await app.resolve(AgentService);
-        agentService.initProfilesFromDisk();
+        await agentService.initProfilesFromDisk();
         if (process.env.MULTI_CONTROLLER === 'true') {
             return;
         }
 
         try {
             await agentService.init();
         } catch (e) {
-            Logger.log(`Failed to start agent, retrying... ${e.message}`, e);
+            Logger.log(`Failed to start agent, retrying... ${e.message as string}`, e);
             try {
                 await ProtocolUtility.delay(1000);
                 await agentService.init();
             } catch (e2) {
-                Logger.log(`Failed to start agent, exiting... ${e2.message}`, e2);
+                Logger.log(`Failed to start agent, exiting... ${e2.message as string}`, e2);
```

Comment:
The linter barks because `e2` implicitly has an `any` type, which means that `e2.message` also implicitly has an `any` type. There's a rule that you can only use certain types in string interpolation.

Given that the transpiler will remove this, I'm not terribly worried about the effect on our packages. We can discuss further, though.